### DISCUSSION TITLE
tuta app (email app)

### DISCUSSION THREAD

**Discussion by Grkrz** [tutao/tutanota#7651] [General]
after the calendar app was added in github it is not possible any more to obtain the email app through obtanium, they are mixed

**Comment by beautiful-orca** [comment]
Edit the Obtainium configuration and add `tutanota-app-tutao-release` to the `Filter APK's by Regular Expression` text field

**Reply by Grkrz** [reply]
Works but you have to remove the obtanium entry you cannot reedit the existing one.
I still think it should be to separate threads for those two apps. 
If a name will chsnge for instance obtanium will not update the APP,
To much od manual work and potential to run the old app

**Reply by beautiful-orca** [reply]
Valid point, release naming has been changed on Aug 15 with 240.240813.0 (Android).
But i think it is likely, that the name wont change again that fast.
Another regex would maybe be more resilient, that filters for `.apk` that contain `tutanota`

**Reply by beautiful-orca** [reply]
Try `.*tutanota.*\.apk$`

**Reply by beautiful-orca** [reply]
I can edit the obtainium entry without problems.

**Reply by CoffeeMan1ac** [reply]
Thank you! Worked for me!

**Comment by 43qcc2cn** [comment]
Just to add that I'm currently filtering the Android Tuta Mail app using the following for "Filter Release Titles by Regular Expression":
`^(\d+).(\d+).(\d+) \(Android\)`

but, having separate repositories for different projects (polyrepo) would be my preference as an end user.

**Comment by DarkCrypt** [comment]
There's no reason for any of this to be labeled as a "bug".

As already stated, there's a few ways to retrieve both apps. The APK filter is all that's necessary and is what I recommend to others. I've discussed this a few times and https://github.com/ImranR98/Obtainium/discussions/2040 is the most recent.

Filter APKs by Regular Expression:
- Tuta Mail: `tutanota-app`
- Tuta Calendar: `calendar`

These 2 filters is all that's necessary.

I'd also recommend to add an optional Trim Version to get rid of the unnecessary text `tutanota-android-release-`. There's no reason to have this added text with the version #.

Trim Version String With ReGex:
- For both apps: `[0-9]+\.[0-9]+\.[0-9]+`
